lib/xboxrt/libc_extensions/wchar.h

```diff
@@ -55,6 +58,21 @@ int mbsinit( const mbstate_t * ps );
 size_t mbrtowc( wchar_t * XBOXRT_RESTRICT pwc, const char * XBOXRT_RESTRICT s, size_t n, mbstate_t * XBOXRT_RESTRICT ps );
 size_t wcrtomb( char * XBOXRT_RESTRICT s, wchar_t wc, mbstate_t * XBOXRT_RESTRICT ps );
 
+wchar_t *wmemset(wchar_t *wcs, wchar_t wc, size_t n);
```

Comment:
Why is there a blank line before this?
Why did we already have prototypes for some functions, but not others?

Reply:
The blank line was left over from visually separating the new stuff back when I wrote this. It's gone now.
The previous prototypes were probably all added on an as-needed basis (which is mostly true for what I've added as well).

Reply:
Fair enough. I'm not sure if order is relevant for something (grouping of features / doxygen / ..) but we can worry about it later.